apps/bfd-pipeline/bfd-pipeline-shared-utils/src/main/java/gov/cms/bfd/pipeline/sharedutils/model/TableEntry.java

```diff
@@ -0,0 +1,32 @@
+package gov.cms.bfd.pipeline.sharedutils.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+/** POJO to contain information about the tables to use in the backfill SQL queries. */
+@Getter
+@Setter
+@AllArgsConstructor
+public class TableEntry {
+  /** The query for the table. */
+  String Query;
```

Comment:
Should be camel-cased

```suggestion
  String query;
```